content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -307,6 +307,18 @@ If expanding underlying storage fails, the cluster administrator can manually re
 4. Re-create the PVC with smaller size than PV and set `volumeName` field of the PVC to the name of the PV. This should bind new PVC to existing PV.
 5. Don't forget to restore the reclaim policy of the PV.
 
+##### User recovery expansion failure
+
+{{< feature-state for_k8s_version="v1.23" state="alpha" >}}
+
+{{< note >}}
+Recovery from expanding PVCs by users is available as an alpha feature since Kubernetes 1.23. The `RecoverVolumeExpansionFailure` feature must be enabled for this feature to work. Refer to the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) documentation for more information.
+{{< /note >}}
+
+In addition to above manual steps Kubernetes if feature gate `RecoverVolumeExpansionFailure` is enabled in the cluster, the user of PVC can retry expansion with smaller size than previously requested value by editing `pvc.spec.resources`.
+This is useful if expansion to a higher value can not succeed because of capacity constraints, in which case user of PVC can retry expansion by specifying a size that is within the capacity limits of underlying storage provider. It should be noted that, while users can specify lower value than what was requested previously in `pvc.spec.resources` but new value must still be higher than `pvc.status.capacity`.
```

Comment:
```suggestion
This is useful if expansion to a higher value can not succeed because of capacity constraints, in which case user of PVC can retry expansion by specifying a size that is within the capacity limits of the underlying storage provider. It should be noted that, while users can specify a lower value than what was requested previously in `pvc.spec.resources` but new value must still be higher than `pvc.status.capacity`.
```

Reply:
Looks like you and @sftim provided conflicting suggested change, if the new change looks good let me know.